src/components/PathwaysList/PathwayModal.tsx

```diff
@@ -49,6 +52,30 @@ const NewPathwayModal: FC<NewPathwayModalProps> = ({ open, onClose }) => {
     [addPathway, closeModal]
   );
 
+  const handleUpdatePathway = useCallback(
+    (event: FormEvent<HTMLFormElement>): void => {
+      event.preventDefault();
+      if (editPath) {
+        if (pathwayNameRef.current?.value) editPath.name = pathwayNameRef.current?.value;
+        if (pathwayDescRef.current?.value !== '')
```

Comment:
Is there a reason the description is checked to be non-empty but the name isn't? I feel like if anything it should be the other way around but in reality both should probably be non-empty

Reply:
Technically, editPath.name has type string and pathwayNameRef.current?.value has type string or undefined, so the conditional is just a way to silence the type checking error from potentially assigning. From a user perspective, I think that the name of the pathway should always be some string, so if the user tries to Update with nothing in the Pathway Name text box, then the previous Pathway Name is maintained to protect against never having a Pathway Name. I can't think of a scenario where the user would actually want their pathway to be completely nameless.

Reply:
With the understanding that, because creating a pathway requires a name, there will always be a previous name.